views/books-show.html

```diff
@@ -0,0 +1,27 @@
+<div class="container">
+  <hr>
+    <div class="clearfix">
+      <div class="col-md-6">
+        <h3><b>Title:</b> <span ng-hide="editing"> {{ booksShowCtrl.book.title }}</span></h3>
```

Comment:
best practice: separate style and semantic meaning - aka no <b>, use a span with a class instead